src/json_v8_renderer.hpp

```diff
@@ -45,27 +45,30 @@ namespace json
 
 struct v8_renderer : mapbox::util::static_visitor<>
 {
-    explicit v8_renderer(v8::Local<v8::Value>& _out) : out(_out) {}
+    explicit v8_renderer(v8::Local<v8::Value> &_out) : out(_out) {}
```

Comment:
I can't find a .clang-format in this repository. Can we use one for consistency?